src/__tests__/index.test.js

```diff
@@ -2,18 +2,18 @@ import path from 'path';
 import fs from 'fs';
 import assert from 'assert';
 import { transformFileSync } from 'babel-core';
-import plugin from '../';
+import plugin from '..';
 
 function trim(str) {
   return str.replace(/^\s+|\s+$/, '');
 }
 
 const fixturesDir = path.join(__dirname, '../__fixtures__');
-fs.readdirSync(fixturesDir).map((caseName) => {
-    it(`${caseName.split('-').join(' ')}`, () => {
+fs.readdirSync(fixturesDir).forEach((caseName) => {
+  it(`${caseName.split('-').join(' ')}`, () => {
```

Comment:
Is this eslint or just not actually mapping here?

Reply:
Its not actually mapping since nothing is being returned. I don't know exactly how it works, but somehow it defines the test functions without returning

Reply:
Oh, its actually invoking `it` in each loop, so that's how it knows to test it

Reply:
👍